composer.lock

```diff
@@ -4,8 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
```

Comment:
You should look at syncing your JSON instead of relying on this. 
This looks to me more of a fallback which you have started to rely on as default. And by looks of it, people are suggesting to use it this way too. 


The world is going to hell.